### PR TITLE
:sparkles: fixed comments in /admin/planningen #299

### LINKED ISSUE #299: admin/planningen fix comments
Comments in admin/planningen/<id> need to be fixed

### PR DESCRIPTION
**Pull request by BrentBlanckaert**
image

Closes #299

### CODE REVIEW

File: frontend/src/pages/admin/planningen/[pid].jsx [hunk 1]
```diff
@@ -172,26 +178,73 @@ export default function AdminTourPage() {
...
           seconds -= minutes * 60;
           time[buildInTour["building"]] = `${minutes}m${seconds}s`;
         }
       }
+      const scheduleCommentResponse =
+        await ScheduleService.getCommentsFromSchedule(planning);
+      for (const comment of scheduleCommentResponse) {
+        const building = await BuildingService.getEntryByUrl(comment.building);
+        const user = await UserService.getEntryByUrl(comment.user);
+        console.log(user);
```
Comment (milachae) [inline frontend/src/pages/admin/planningen/[pid].jsx:239]: remove this.

File: frontend/src/pages/admin/planningen/[pid].jsx [hunk 2]
```diff
@@ -172,26 +178,73 @@ export default function AdminTourPage() {
...
+        comments.push({
+          comment: comment.text,
+          building: building.nickname,
+          user: `${user.first_name} ${user.last_name}`,
+          last_update: new Date(comment.updated_at),
+        });
+      }
+      console.log(comments);
```
Comment (milachae) [inline frontend/src/pages/admin/planningen/[pid].jsx:247]: remove this

**Review by milachae — CHANGES_REQUESTED** [review]
Looks good, remove the console.logs

**Review by milachae — APPROVED** [review]

**Comment by milachae** [comment]
@BrentBlanckaert 
The button on the planning page forwards you to /planning/<id> and not /planningen/<id>, 2 options here:
* change the detailed page to planning
* change the main page to planningen and change the title in the navbar to planningen

**Comment by BrentBlanckaert** [comment]
> @BrentBlanckaert The button on the planning page forwards you to /planning/ and not /planningen/, 2 options here:
> 
> * change the detailed page to planning
> * change the main page to planningen and change the title in the navbar to planningen

This should be fixed in the PR #307

**Pull request merged**